package.json

```diff
@@ -1,9 +1,11 @@
 {
-  "name": "forta-agent-starter",
+  "name": "maker-bot",
   "version": "0.0.1",
-  "description": "Forta Agent Typescript starter project",
+  "description": "MakerDao Dai bridge solvency checker",
   "chainIds": [
```

Comment:
Keep in mind that we also need to define the `repository` (the link should be the one of the main branch after the PR merge)